dataactbroker/handlers/fileHandler.py

```diff
@@ -1443,13 +1464,18 @@ def list_submissions(page, limit, certified, sort='modified', order='desc', d2_s
     if not options.get(sort):
         sort = 'modified'
 
-    model = options[sort]['model']
-    col = options[sort]['col']
+    order_by = getattr(options[sort]['model'], options[sort]['col'])
```

Comment:
Can we change this variable name? It seems like bad practice to have a variable named the same thing as a function we're using on it (farther down)